mongodbatlas/projects.go

```diff
@@ -99,6 +99,10 @@ type TeamsAssigned struct {
 	TotalCount int       `json:"totalCount"`
 }
 
+type ProjectOptions struct {
```

Comment:
[nit] let's make this create specific 
```suggestion
type CreateProjectOptions struct { // or any other similar name 
```